i18n/locales/source/course_content/2023/csc-mappinglandmarks-2023.json

```diff
@@ -51,26 +51,7 @@
   },
   "https://studio.code.org/s/csc-mappinglandmarks-2023/lessons/1/levels/6": {
     "short_instructions": "Assign a `moving with arrow keys` behavior to the tour guide to move them around the map. ",
-    "long_instructions": "**Get the tour guide moving around the map! Assign a behavior.**\n\n-   Behaviors are actions the sprite performs over and over, like an animation.\n-   Add a <code style=\"color:black; background-color:#5ED5DD\">sprite begins moving with arrow keys</code> behavior block.\n-   Click <code style=\"color:black; background-color:#FFBD46\">Run</code> and test it out by pressing your arrow keys.",
```

Comment:
Chunk of removed strings here. Not sure if it this should have been deleted? 🤷‍♂️